demos/toolbar/default-flexible-toolbar.html

```diff
@@ -25,36 +25,11 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <style>
-      .mdc-toolbar-demo {
-        margin: 0px;
-      }
-      .mdc-toolbar__row:first-child::after {
-        background-image: url("../images/4-3.jpg");
-        background-size: cover;
-        background-position: center;
-      }
-      .demo-paragraph {
-        margin: 0px;
-        padding: 20px 28px;
-      }
-      @media (max-width: 599px) {
-        .demo-paragraph {
-          padding: 16px;
-        }
-      }
-      footer {
-        position: fixed;
-        bottom: 20px;
-        width: 100%;
-        text-align: center;
-        color: red;
-        background-color: rgba(0, 0, 0, 0.8);
-      }
     </style>
   </head>
-  <body class="mdc-typography mdc-toolbar-demo">
-    <header class="mdc-toolbar mdc-toolbar--flexible mdc-toolbar--flexible-default-behavior">
-      <div class="mdc-toolbar__row">
+  <body class="mdc-typography">
+    <header class="mdc-toolbar mdc-toolbar--flexible mdc-toolbar--flexible-default-behavior demo-toolbar">
```

Comment:
should this have the class demo-toolbar--flexible to be consistent with the custom toolbar?

Reply:
on second thought...its just the demo so nevermind